pkg/localstore/sampler.go

```diff
@@ -232,7 +232,7 @@ func (db *DB) ReserveSample(
 		Items: sampleItems,
 		Hash:  swarm.NewAddress(hash),
 	}
-	logger.Info("sampler done", "stats", stat, "sample", sample)
+	logger.Info("sampler done", "stats", &stat, "sample", sample)
```

Comment:
Instead, I suggest changing the `String()` to a method without a pointer receiver.